ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/evpn_import_pruning.schema.yml

```diff
@@ -5,12 +5,10 @@ type: dict
 keys:
   evpn_import_pruning:
     documentation_options:
-      filename: Input Variables
       table: EVPN Settings
     description: |
-      Enable VPN import pruning (Min. EOS 4.24.2F)
-      The Route Target extended communities carried by incoming VPN paths will
-      be examined. If none of those Route Targets have been configured for import,
-      the path will be immediately discarded
+      Enable VPN import pruning (Min. EOS 4.24.2F).
```

Comment:
Should the schema have keys for `eos_introduced`, and `avd_introduced`?

Reply:
It would be hard to maintain the mapping between features, EOS and various platforms.